src/benchmarks/micro/libraries/System.Runtime.Numerics/Perf.BigInteger.cs

```diff
@@ -133,16 +172,24 @@ public class BigIntegers
             public BigIntegers(int[] bitCounts)
             {
                 _bitCounts = bitCounts;
+                _description = $"{string.Join(",", _bitCounts)} bits";
```

Comment:
Thank you for keeping the description the same for existing PRs (the result of `ToString` is used as part of benchmark ID which should never change: https://github.com/dotnet/performance/blob/main/docs/microbenchmark-design-guidelines.md#benchmarks-are-immutable) 👍